Doc/library/sqlite3.rst

```diff
@@ -1007,9 +1007,15 @@ Cursor Objects
 
       *sql_script* must be a :class:`string <str>`.
 
-      Example:
+      Example::
 
-      .. literalinclude:: ../includes/sqlite3/executescript.py
+         cur.executescript("""
```

Comment:
In contrast with the tutorial, examples in the reference can assume familiarity. I believe it is ok to drop the import/connect setup, but I can be convinced to include it.

Reply:
I'm generally sympathetic to the argument of keeping them minimal and assuming a basic level of familiarity. On the other hand, Reference docs should also prioritize clarity above all, and at least to me it isn't great form to leave what `cur` is completely implicit—at least a comment to the affect of `# cur is a sqlite3.Cursor object` would help alleviate that.

Reply:
Yes, I was worried about this. Let's introduce a comment.

Reply:
```suggestion
         # cur is an sqlite3.Cursor object
         cur.executescript("""
```